components/webui/imports/api/ingestion/server/publications.js

```diff
@@ -45,6 +42,13 @@ let compressionJobsRefreshTimeout = null;
  */
 let statsRefreshInterval = null;
 
+/**
+ * @type {string}
+ */
+let lastUpdateDate = new Date().toISOString()
+    .slice(0, CONST_FOR_DATE_FORMAT)
+    .replace("T", " ");
+
```

Comment:
As discussed in `components/webui/imports/api/ingestion/server/CompressionDbManager.js`, instead of tracking the time as a string, can we store it as a numerical timestamp instead? 
If so, we can name it `lastUpdateTimestampSeconds`